jax/_src/numpy/ufuncs.py

```diff
@@ -2495,12 +2495,20 @@ def floor_divide(x1: ArrayLike, x2: ArrayLike, /) -> Array:
   x1, x2 = promote_args_numeric("floor_divide", x1, x2)
   dtype = dtypes.dtype(x1)
   if dtypes.issubdtype(dtype, np.unsignedinteger):
-    return lax.div(x1, x2)
+    result = lax.div(x1, x2)
+    result = lax.select(_broadcast_to(x2 == 0, result.shape),
+                        lax.full_like(result, fill_value=0),
+                        result)
+    return result
```

Comment:
Using `_where` would make this more succinct. I think this should work:
```suggestion
    return _where(x2 == 0, 0, lax.div(x1, x2))
```

Reply:
Thank! Done.